.travis.yml

```diff
@@ -1,16 +1,15 @@
+sudo: false
 language: python
 env:
-  - REQUESTS_VERSION=2.1.0
-  - REQUESTS_VERSION=2.8.1
-  - REQUESTS_VERSION=2.9.1
-python:
-  - "2.6"
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-install:
-  - if [[ $TRAVIS_PYTHON_VERSION == 2.6* ]]; then export PYTEST_ADDOPTS='-p no:pylint'; fi
-  - pip install -r requirements.txt
-  - pip install --upgrade --force-reinstall requests==$REQUESTS_VERSION
-script: py.test
+    - TOXENV=py27-requests-current
```

Comment:
Minor: does Travis have any syntax for syncing this list of envs to run with tox.ini? Too bad it has to be in 2 places. But it's not too bad.